Client/Frontend/Fakespot/Views/FakespotReviewQualityCardView.swift

```diff
@@ -79,7 +79,7 @@ final class FakespotReviewQualityCardView: UIView, Notifiable, ThemeApplicable {
         label.numberOfLines = 0
         label.adjustsFontForContentSizeCategory = true
         label.clipsToBounds = true
-        label.font = DefaultDynamicFontHelper.preferredFont(withTextStyle: .body,
```

Comment:
This string (`.Shopping.ReviewQualityCardSubHeadlineLabel`) needs to be updated and part of it needs to be bold.